challenges/ll_find_loop/ll_find_loop.py

```diff
@@ -0,0 +1,48 @@
+class Node:
+    def __init__(self, value, next=None):
+        self.val = value
+        self._next = next
+
+    def _has_loop(self):
```

Comment:
This method should be on the `LinkedList` class instead of the `Node` class. Or rather, you should put the contents of this method into the `has_loop()` method that is already on your `LinkedList` class.